presto-main/src/main/java/io/prestosql/operator/TaskContext.java

```diff
@@ -375,6 +382,18 @@ public int getFullGcCount()
         return toIntExact(max(0, endFullGcCount - startFullGcCount));
     }
 
+    public DynamicFilterSourceConsumer createDynamicFiltersConsumer(String source, int partitionsCount)
```

Comment:
I think we should have similar mechanism as `LocalDynamicFiltersCollector` that collects dynamic filters from multiple joins. I think we might even reuse `LocalDynamicFiltersCollector` for that.